jep/227/README.adoc

```diff
@@ -210,8 +209,9 @@ Exception Details:
 
 === Providing binary compatibility for all plugins implementing `SecurityRealm`
 
-Some plugins like `ldap` which implement `SecurityRealm` pose a special problem.
+Some plugins like `sfee` which implement `SecurityRealm` pose a special problem.
```

Comment:
The `sfee` plugin is removed from distribution.  The stats data shows only 6 installations, 1.301 and earlier.  Is there a plugin example that poses this problem that is not removed from distribution? 

If not, it may be worth noting the state of that plugin.
```suggestion
Some plugins like `sfee` which implement `SecurityRealm` pose a special problem.
```

Reply:
This particular plugin may not be important, it was just an example of source code that is particularly problematic to migrate.

Reply:
I was unable to find the source code for that plugin in my searches.  Wouldn't it be better to use an example plugin where source code is readily available?

Reply:
https://github.com/jenkinsci/sfee-plugin